### ISSUE TITLE
With the new look of YouTube, NVDA does not see some headings in search results

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#7611]
### Description:
If the Microsoft Edge browser is in use, with the new look/interface of the YouTube website, NVDA seams to fail to navigate to and read correctly some headings, specially those from level 3 in the search results. When I use H or 3 to navigate to the results from a search in YouTube, NVDA says only "blank" for search results that are for videos (it behaves as it should if the search result is for a channel, it seams).

### Steps to reproduce:
1. Open Microsoft Edge
2. Browse to www.youtube.com
3. Type a search term in the search edit field and press ENTER to execute the search.
4. Use H or 3 to navigate to the headings that comprise the list of search results.

### Expected behavior:
NVDA should read each search result as we navigate to it with H or 3.

### Actual behavior:
 NVDA says "blank" for each search result for a video (it seams to behave normally if the search result is for a channel).

### System configuration:
- Operating system: Windows 10 Pro version 1703 (build 15063.608)), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-14432,821b31df, installed, in bulgarian.
- Microsoft Edge version: 40.15063.0.0, Microsoft EdgeHTML 15.15063.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: Palit GeForce GT 630, 1024MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

### Notes:
In theory, the new look/interface of YouTube should be active for all users. If you find a "Settings" button somewhere on top of the page in YouTube even when you are not logged in your Google account, that means that the new look/interface is active for you. Though now I've noticed that the "Settings" button seams to be hard to find. The easiest way to find it is with the elements list in the NVDA "next" snapshots, where there is a way to list all buttons. But that is just to check if the new look/interface is active for us or not. To activate that new look/interface and to check if it is active or not, go to the following web page of YouTube: [https://www.youtube.com/new](url). Though it seams that the new look/interface is now active for all users.

**Comment by ehollig** [comment]
Possibly related: #7539
I can replicate this issue on Youtube in Edge. CC @michaelDCurran

**Comment by feerrenrut** [comment]
Thanks for the detail included when raising this issue!
I haven't been able to reproduce this with NVDA version `master-14436,b73a9583`.
But I can with NVDA version `next-14463,7dca66c4`

My guess is that something currently incubating is causing the problem. Though looking through the incubating issues nothing jumps out.

A diff of master and next shows that there is a change in `source/NVDAObjects/IAccessible/MSHTML.py` created in commit e1198327
which renames a function to `def old_get_shouldAllowIAccessibleFocusEvent(self):`

These seem related to https://github.com/nvaccess/nvda/issues/5676 (Found through the commit message where e1198327 is merged to next)

**Comment by feerrenrut** [comment]
Seems I wasn't thinking straight in my last comment, `MSHTML.py` relates to IE not Edge. While these commits need to be sorted out, it isn't the change we are looking for.

**Comment by gdata1** [comment]
If the 7538 problem is closed. 'll answer here.
I have this problem now.

**Comment by feerrenrut** [comment]
Considering the difference between next and master the last time I investigated, I think we should try to track down why this is the case. Giving this a P1 to investigate further.

**Comment by dkager** [comment]
Can reproduce with master-14883,c83231ec. Interestingly, the object name is set but not announced or brailled. You can also not arrow through the heading text.

**Comment by leonardder** [comment]
This issue has a priority of p1 while abandoned for more than half a year now. Does this priority still apply?

**Comment by k-kolev1985** [comment]
The issue seams to be solved now. It was probably a bug in Edge itself. We may close the issue I think. I think it was fixed in Windows 10 version 1809.

**Comment by gdata1** [comment]
yes

**Comment by ehollig** [comment]
Closing as requested. If it comes up again, we can reopen this issue. Closing as worksforme
